src/tests/ftest/erasurecode/aggregation.py

```diff
@@ -31,8 +31,8 @@ def test_ec_aggregation_disabled(self):
         # Disable the aggregation
         self.pool.set_property("reclaim", "disabled")
         self.pool.connect()
-        print("pool_percentage Before = {} ".format(
-            self.pool.pool_percentage_used()))
+        self.log.info("pool_percentage Before = {} ".format(
```

Comment:
(pylint-logging-format-interpolation) Use lazy % formatting in logging functions